packages/engine/Source/Scene/Terrain.js

```diff
@@ -0,0 +1,182 @@
+import Check from "../Core/Check.js";
+import Event from "../Core/Event.js";
+import createWorldTerrainAsync from "../Core/createWorldTerrainAsync.js";
+
+function handleError(errorEvent, error) {
```

Comment:
Can we move the helper functions down, so the [constructor is at the top of the file](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CodingGuide/README.md#put-the-constructor-function-at-the-top-of-the-file)?